src/applications/gqrx/remote_control.cpp

```diff
@@ -429,19 +429,24 @@ void RemoteControl::setMode(int mode)
 void RemoteControl::setNewRemoteFreq(qint64 freq)
 {
     qint64 delta = freq - rc_freq;
+    // it's an ugly but I hope this will be fixed with the issue #218
+    int FilterBW_half = std::min<int>(bw_half, 100000 /* WFM Wide */);
```

Comment:
I will give it a try later when I have time, but I doubt I will accept it in its present form. It behaves differently and for sure it may be better in some scenarios, but it is worse in other scenarios.